### PR TITLE
(40) Show individual place from the leaderboard

### PR DESCRIPTION
**Pull request by CristinaRO**
## Changes in this PR
- The thumbnails on the leaderboard link to bigger and more detailed views of the image

### After
Screenshot 2022-04-25 at 15 54 53

### CODE REVIEW

**Comment by CristinaRO** [comment]
[Show individual place from the leaderboard](https://trello.com/c/EWpE36BG/40-show-individual-place-from-the-leaderboard)

File: app/controllers/places_controller.rb [hunk 1]
```diff
@@ -11,6 +11,10 @@ def leaderboard
     @leaderboard = LeaderboardPresenter.new
   end
 
+  def show
+    @place = PlacePresenter.new(Place.find(params[:id]))
```
Comment (pezholio) [inline app/controllers/places_controller.rb:15]: I think I'd prefer these urls to have a slug, but this can come later

**Review by pezholio — APPROVED** [review]
LGTM 👍

File: app/controllers/places_controller.rb (hunk 1 above)
Comment (CristinaRO) [reply app/controllers/places_controller.rb:15]: It's true that this is how they were done in the original version, will have a look.

Comment (pezholio) [reply app/controllers/places_controller.rb:15]: https://github.com/norman/friendly_id might be worth a look

**Pull request merged**